javascript/node/selenium-webdriver/index.js

```diff
@@ -490,6 +490,18 @@ class Builder {
     return this;
   }
 
+  /**
+   * Sets the {@link ie.ServiceBuilder} to use to manage the geckodriver
+   * child process when creating IE sessions locally.
+   *
+   * @param {ie.ServiceBuilder} service the service to use.
+   * @return {!Builder} a self reference.
+   */
+  setIeService(service) {
+    this.ieService_ = service;
```

Comment:
`this.ieService_` should be initialized to null in the constructor.

Reply:
Done!